### PR TITLE
Comply with awesome list guidelines

### PR DESCRIPTION
**Pull request by kevinkassimo**
__WIP__. See https://github.com/sindresorhus/awesome/pull/1531

- Remove Travis build status badge
- Add Deno project description
- Update module description (All start with a noun or a verb (no prefix a/an/the and no -s suffix for verbs))
- Add CC0 license
- Add contributing guide

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,56 +38,56 @@
...
-- [oak](https://github.com/oakserver/oak) - A middleware framework for Deno's net server.
+- [oak](https://github.com/oakserver/oak) - Middleware framework for Deno's net server.
 - [path](https://github.com/denoland/deno_std/tree/master/fs/path) - Deno Path Manipulation Libraries.
 - [pogo](https://github.com/sholladay/pogo) - Server framework for Deno
 - [postgres](https://github.com/bartlomieju/deno-postgres) - Driver for PostgreSQL database.
-- [qs](https://github.com/denolib/qs) - A querystring parser with nesting support.
-- [textproto](https://github.com/denoland/deno_std/tree/master/textproto)
-- [watch](https://github.com/jinjor/deno-watch) - A file watcher.
-- [wu-diff-js](https://github.com/bokuweb/wu-diff-js) - A diff library to compute differences between two slices using wu(the O(NP)) algorithm.
+- [qs](https://github.com/denolib/qs) - Querystring parser with nesting support.
```
Comment (sindresorhus) [inline README.md]:
```suggestion
- [qs](https://github.com/denolib/qs) - Query string parser with nesting support.
```

File: README.md [hunk 2]
```diff
@@ -34,56 +38,56 @@
...
-- [deno_ls_plugin](https://www.npmjs.com/package/deno_ls_plugin) - a TypeScript plugin which will allow TypeScript outside of Deno to resolve modules in a similar way to the way they are resolved inside of Deno.
-- [deno_init](https://github.com/zhmushan/deno_init) - Quickly initialize a Deno project.
-- [dpm](https://github.com/BoltDoggy/deno#dpm) - Deno Package Manager, install global command for deno. like denoget.
-- [dvm](https://github.com/justjavac/dvm) - Deno Version Manager: manage multiple active deno versions.
-- [elm-live-reload](https://github.com/jinjor/deno-playground/tree/master/elm-live-reload) - An elm live reloader written in Deno.
+- [deno_ls_plugin](https://www.npmjs.com/package/deno_ls_plugin) - TypeScript plugin which will allow TypeScript outside of Deno to resolve modules in a similar way to the way they are resolved inside of Deno.
+- [deno_init](https://github.com/zhmushan/deno_init) - Initialize a Deno project quickly .
+- [dpm](https://github.com/BoltDoggy/deno#dpm) - Install global command for deno (Deno Package Manager), like denoget.
+- [dvm](https://github.com/justjavac/dvm) - Manage multiple active deno versions (Deno Version Manager).
```
Comment (sindresorhus) [inline README.md]:
```suggestion
- [dvm](https://github.com/justjavac/dvm) - Manage multiple active Deno versions (Deno Version Manager).
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+- Make an individual pull request for each suggestion.
+- Use the following format: `[package](link) - Description.`
+- Additions should be added to the bottom of the relevant category.
+- Link to the GitHub repo.
+- Keep descriptions short and simple, but descriptive.
+- Start the description with a capital and end with a full stop/period.
+- Start the description with either a verb or a noun phrase.
+- Don't start the description with `A`, `An` or `The`.
+- Don't add `-s` suffix if the description starts with a verb.
+- Avoid `(for) Deno` in the description unless necessary. Stylize as `Deno` instead of `deno`.
```
Comment (motss) [inline CONTRIBUTING.md]: I realized that `deno fmt .` actually formats/ rewrites all the `Deno` word in `.md` files (can't recall what else file formats it affects) to `deno`.

@sindresorhus Can we check with @ry and fellow contributors on this? Which name should the community use?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# Contribution Guidelines
+
+Ensure your pull request adheres to the following guidelines:
+
+- If you submit a project that is similar to an existing project in the list, argue how it's different or better.
+- Suggested packages should be tested and documented.
+- Make an individual pull request for each suggestion.
+- Use the following format: `[package](link) - Description.`
+- Additions should be added to the bottom of the relevant category.
```
Comment (motss) [inline CONTRIBUTING.md]: Does it mean all newly added modules will be at the bottom instead of sorting them alphabetically?

Comment (zekth) [reply CONTRIBUTING.md]: Alphabetical ordering is better for me.

**Comment by zekth** [comment]
Maybe is it worth is to add `PR_TEMPLATE` too ?

**Comment by kt3k** [comment]
Closing in favor of #123. Thank you @kevinkassimo for the prior effort.

**Pull request closed without merging**